verification/verify_test.go

```diff
@@ -135,3 +150,59 @@ func TestVerify(t *testing.T) {
 	assert.Equal(err, verifier.err)
 	assert.Nil(res)
 }
+
+func TestPixels(t *testing.T) {
+	ffmpeg.InitFFmpeg()
+
+	assert := assert.New(t)
+
+	p, err := Pixels("foo")
+	assert.EqualError(err, "No such file or directory")
+	assert.Equal(int64(0), p)
+
+	// Assume that ffmpeg.Transcode3() returns the correct pixel count so we just
+	// check that no error is returned
+	p, err = Pixels("test.flv")
+	assert.Nil(err)
+	assert.NotZero(p)
+}
+
+func TestVerifyPixels(t *testing.T) {
+	ffmpeg.InitFFmpeg()
+
+	require := require.New(t)
+	assert := assert.New(t)
+
+	// Create memory session and save a test file
+	bos := drivers.NewMemoryDriver(nil).NewSession("foo")
+	data, err := ioutil.ReadFile("test.flv")
```

Comment:
Can we make this a relative path, eg "../server/test.flv" ? This would avoid (re-)adding the large "test.flv" blob, which Git doesn't really handle well.

Reply:
done in 0670fe4097f2d7312bdc818f2aa5d041ec565b80